spec/struct.dd

```diff
@@ -1106,6 +1106,26 @@ $(H3 $(LNAME2 field-init, Field initialization inside a constructor))
 
 $(H2 $(LEGACY_LNAME2 StructCopyConstructor, struct-copy-constructor, Struct Copy Constructors))
 
+    $(P $(RED Warning:) The plan is for
+    $(RELATIVE_LINK2 struct-copy-constructor, copy constructors) to replace
+    $(RELATIVE_LINK2 struct-postblit, postblit constructors), leaving postblit
+    constructors only in legacy code. However, because the compiler hooks in
+    druntime which deal with dynamic arrays and associative arrays have not yet
+    all been updated to support copy constructors properly
+    $(HTTPS github.com/dlang/dmd/issues/20970, issue #20970), any type that might
```

Comment:
```suggestion
    ($(HTTPS github.com/dlang/dmd/issues/20970, issue #20970)), any type that might
```

I'd put parentheses around this, because it doesn't fit in the sentence the way it is worded now.

Reply:
Thanks for catching that. It's what I had intended, but I missed it because of the set of parens that's there for the ddoc macro.

Fixed.